RecoTracker/IterativeTracking/python/ElectronSeeds_cff.py

```diff
@@ -16,8 +16,8 @@
     oldClusterRemovalInfo = cms.InputTag("initialStepSeedClusterMask")
 )
 eras.trackingPhase1PU70.toModify(pixelPairStepSeedClusterMask, oldClusterRemovalInfo = "highPtTripletStepSeedClusterMask")
-# This is a pure guess to use lowPtTripletStep for phase1 here instead of the pixelPair in Run2 configuration
-lowPtTripletStepSeedClusterMask = seedClusterRemover.clone(
+# This is a pure guess to use detachedTripletStep for phase1 here instead of the pixelPair in Run2 configuration
+detachedTripletStepSeedClusterMask = seedClusterRemover.clone(
```

Comment:
Once we have a baseline in IB or release EGAMMA should start to have a look to all this...